.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,54 @@
+name: build and snyk
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
+    branches: 
+      - main
+
+jobs:
+  build:
```

Comment:
build doesn't require p_r_target - only snyk does

Reply:
addressed and will do similar change for gradle plugin PRs.